format/Flight.proto

```diff
@@ -259,8 +259,10 @@ message FlightInfo {
   FlightDescriptor flight_descriptor = 2;
 
   /*
-   * A list of endpoints associated with the flight. To consume the whole
-   * flight, all endpoints must be consumed.
+   * A list of endpoints associated with the flight. To consume the
+   * whole flight, all endpoints (and hence all Tickets) must be consumed.
+   *
+   * In other words, multiple endpoints provide partitioning.
```

Comment:
That's a very good point. I've reworded things here, how does it look now?

Reply:
I'm confused by your comment here a little bit. Flight *can* do partitioning. It is done by returning more than one ticket for a particular flight. (I agree that locations aren't about partitioning.)

Reply:
I suppose it's because an application can implement partitioning via Flight, but Bryan has seen people thinking this means that using Flight will automatically partition/distribute data.

Reply:
What I meant was that Flight doesn't have built-in logic to actively partition a given data set. It's up to the service implementation to assign partitions to a given ticket.